src/software/simulated_tests/simulated_er_force_sim_test_fixture.cpp

```diff
@@ -178,8 +179,9 @@ void SimulatedErForceSimTestFixture::runTest(
     const Duration simulation_time_step =
         Duration::fromSeconds(1.0 / SIMULATED_CAMERA_FPS);
 
-    std::shared_ptr<ErForceSimulator> simulator(
-        std::make_shared<ErForceSimulator>(field_type, create2021RobotConstants()));
+    auto realism_config = ErForceSimulator::createDefaultRealismConfig();
+    std::shared_ptr<ErForceSimulator> simulator(std::make_shared<ErForceSimulator>(
+        field_type, create2021RobotConstants(), realism_config));
```

Comment:
@itsarune Is the plan to support realistic sim in simulated tests later?

Reply:
I missed this sorry, rn there is no plan (unless you think it's necessary)